### PR TITLE
update release-3.1 images for tag v3.1.0

### PR DESCRIPTION
**Pull request by zryfish**
Signed-off-by: Jeff <jeffzhang@yunify.com>

Update images list used by KubeSphere v3.1.0

### CODE REVIEW

**Comment by daniel-hutao** [comment]
🥇

File: scripts/images-list.txt [hunk 1]
```diff
@@ -39,28 +38,24 @@ csiplugin/csi-resizer:v0.4.0
...
+kubesphere/kubectl:v1.19.0                 
 redis:5.0.5-alpine                              
 alpine:3.10.4                                   
 haproxy:2.0.4                                   
-mysql:8.0.11                                    
 nginx:1.14-alpine                               
 minio/minio:RELEASE.2019-08-07T01-59-21Z        
 minio/mc:RELEASE.2019-08-07T23-14-43Z           
 mirrorgooglecontainers/defaultbackend-amd64:1.4 
 kubesphere/nginx-ingress-controller:v0.44.0
```
Comment (zackzhangkai) [inline scripts/images-list.txt:54]: ingress controller has changed to v0.35.0

**Comment by junotx** [comment]
`kubesphere/ks-alerting-migration:v3.1.0` is required to upgrade alerting

File: scripts/images-list.txt [hunk 2]
```diff
@@ -126,12 +123,14 @@ kubesphere/notification:v2.1.2
 kubesphere/alert-adapter:v3.0.0             
 kubesphere/alerting-dbinit:v3.0.0           
 kubesphere/alerting:v2.1.2
```
Comment (junotx) [inline scripts/images-list.txt:125]: These images for old alerting can be removed

**Pull request merged**